test/functional/ui/output_spec.lua

```diff
@@ -230,4 +231,26 @@ describe("shell command :!", function()
       ]])
     end)
   end)
+
+  if iswin() then
+    describe('powershell', function()
+      local screen
+      before_each(function()
+        clear()
+        set_shell_powershell()
+        screen = Screen.new(30, 6)
+        screen:attach()
+      end)
+      after_each(function()
+        screen:detach()
+      end)
+
+      it('supports literal strings', function()
```

Comment:
`screen:detach()` isn't needed so `after_each` isn't needed.

In general it's better to avoid nested `before_each`, so it would be better to put the screen setup directly at the start of the `it()`. Then we don't need to mentally compose the nested `before_each` blocks to understand the test steps.

Also the `clear()` should be removed.